java/client/src/main/java/glide/connectors/handlers/ReadHandler.java

```diff
@@ -33,6 +33,7 @@ public void channelRead(@NonNull ChannelHandlerContext ctx, @NonNull Object msg)
     @Override
     public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws Exception {
         Logger.log(ERROR, "read handler", () -> "=== exceptionCaught " + ctx + " " + cause);
```

Comment:
Please update this too
```suggestion
        Logger.log(ERROR, "read handler", () -> "exceptionCaught " + ctx);
```